payments/vipps/redirect.md

```diff
@@ -97,46 +83,253 @@ sequenceDiagram
   Browser->>Merchant: start purchase (pay with VIPPS)
   activate Merchant
 
-  Merchant->>PayEx_FrontEnd: POST <Create  Vipps payment>
+  Merchant->>Swedbank_Pay_FrontEnd: POST <Create  Vipps payment>
```

Comment:
1. Instead of underscores, just use `PascalCase` in the diagrams.
2. Also use an alias to get a more human readable name in the presentation.
3. I think we can merge `SwedbankPayFrontEnd` and `SwedbankPayBackEnd` into one actor to simplify the diagram.

```suggestion
  Merchant->>SwedbankPayFrontEnd: POST <Create  Vipps payment>
```